Python/Aws/retryMechanism.py

```diff
@@ -0,0 +1,31 @@
+from pg import DB, InternalError
```

Comment:
```suggestion
What is this pg module?
```

Reply:
`pg`  refers to the `PyGreSQL` driver. `PyGreSQL` is an open-source Python module that interfaces to a PostgreSQL database. It embeds the PostgreSQL query library to allow easy use of the powerful PostgreSQL features from a Python script.

For more information read [this](http://www.pygresql.org/)